test/integration-tests/lib/template_provider.rs

```diff
@@ -149,6 +149,12 @@ impl TemplateProvider {
         client.generate_to_address(101, &address)?;
         Ok(())
     }
+
+    pub fn no_of_blocks(&self) -> Result<i64, corepc_node::Error> {
```

Comment:
What is "no_of_blocks"? please use full clear names.
I also dont see this is used anywhere? and it is missing docs.